internal/bitset/bitset.go

```diff
@@ -0,0 +1,90 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+// Third-party implementation: https://github.com/willf/bitset/blob/master/bitset.go
```

Comment:
The way you wrote it makes it part of the copyright header. I guess this is best placed about the `type Bitset` structure (but not mixed with the documentation). And this isn't the "third party implementation", this is where the code was copied from, wasn't it? Then we should maybe say that. Something like "code was inspired from github.com/willf/bitset".